biogeochem/EDPhysiologyMod.F90

```diff
@@ -661,10 +661,31 @@ subroutine phenology( currentSite, bc_in )
     endif
     !
     ! accumulate the GDD using daily mean temperatures
-    if (bc_in%t_veg24_si .gt. tfrz) then
+    ! Don't accumulate GDD during the growing season (that wouldn't make sense)
+    if (bc_in%t_veg24_si .gt. tfrz.and. currentSite%cstatus == phen_cstat_iscold) then
        currentSite%grow_deg_days = currentSite%grow_deg_days + bc_in%t_veg24_si - tfrz
     endif
     
+    !this logic is to prevent GDD accumulating after the leaves have fallen and before the 
+    ! beginnning of the accumulation period, to prevend erroneous autumn leaf flushing. 
+    if(model_day_int>365)then !only do this after the first year to prevent odd behaviour
+
+    if(currentSite%lat .gt. 0.0_r8)then !Northern Hemisphere                                           
+      ! In the north, don't accumulate when we are past the leaf fall date.
+      ! Accumulation starts on day 1 of year in NH.  
+      ! The 180 is to prevent going into an 'always off' state after initialization
+      if( model_day_int .gt. currentSite%cleafoffdate.and.hlm_day_of_year.gt.180)then !
```

Comment:
@rosiealice , I think it would be nice to give 180 an explicit constant name as well.

Reply:
Yes, although the 'start of counting date' might be more tractable and we could key the 180 off that?

Reply:
sounds good